### ISSUE TITLE
System.NullReferenceException for users with roles Front desk and Housekeeping

### ISSUE THREAD

**Issue by pgrm** [apaleo/oss-traces#91]
I created a test user with the roles Front desk and Housekeeping and got the following exception when trying to access oss-traces:

```
2020-03-10 16:08:59.0810||2e86e1a5-03e3-4b6c-8011-7d70498128fe|||Microsoft.AspNetCore.Components.Server.Circuits.CircuitHost|ERROR|Unhandled exception in circuit '46GrciPBFlAhHh158-SKnCaqYJydj_oyJeOVovpAi5A'. System.NullReferenceException: Object reference not set to an instance of an object.
   at Traces.Web.Services.Apaleo.ApaleoUserClaimValidatorService.AssertClaim(String queryParameter, String claimType) in /app/src/Traces.Web/Services/Apaleo/ApaleoUserClaimValidatorService.cs:line 40
   at Traces.Web.App.OnInitialized() in /app/src/Traces.Web/App.razor:line 31
   at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()
```

The weirdest part, it worked once on staging, but after trying few more times with switching users, .. it's also broken on staging now.

**Comment by pgrm** [comment]
@haydar-metin so this can be closed, yes?

**Comment by haydar-metin** [comment]
should be fixed once we move to client side blazor 3.2 in the near future
